src/tools/gophish_export.py

```diff
@@ -94,25 +116,30 @@ def pull_gophish_groups(api, assessment_id):
 
 
 def campaignControl(api, assessment_id):
-    """Control the campaign importing of an assessment to DB."""
-    # STAND ALONE
+    """Control the campaign importing of an assessment to DB.
+
+    Results are saved to a json file.
```

Comment:
I would recommend renaming this function "export_campaigns" and updating the description appropriately.  Again `Results are saved to a json file.` is unnecessary here.

Reply:
Removed comment regarding json in fa6ca50.